metadata.json

```diff
@@ -38,18 +38,17 @@
     {
```

Comment:
Could you please also add the new Puppet versions being supported? 🙏

Reply:
Ooops, missed that!

Reply:
Build is failing.

I'm afraid that supporting the module on an unsupported version of Puppet running on an unsupported version of Ruby is a waste of resources :confused: